Content.Server/Ghost/Roles/Components/GhostRoleMobSpawnerComponent.cs

```diff
@@ -10,17 +10,23 @@ namespace Content.Server.Ghost.Roles.Components
     [Access(typeof(GhostRoleSystem))]
     public sealed partial class GhostRoleMobSpawnerComponent : Component
     {
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("deleteOnSpawn")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("deleteOnSpawn")]
         public bool DeleteOnSpawn = true;
 
-        [ViewVariables(VVAccess.ReadWrite)] [DataField("availableTakeovers")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("availableTakeovers")]
         public int AvailableTakeovers = 1;
 
         [ViewVariables]
         public int CurrentTakeovers = 0;
 
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("prototype", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
```

Comment:
```suggestion
        [DataField(customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
```
Get this one too while you're here.
Check if the customTypeSerializer is still needed too. I'm not sure offhand.